velox/exec/RowContainer.cpp

```diff
@@ -282,7 +282,7 @@ char* RowContainer::initializeRow(char* row, bool reuse) {
     auto rows = folly::Range<char**>(&row, 1);
     freeVariableWidthFields(rows);
     freeAggregates(rows);
-  } else if (rowSizeOffset_ != 0 && checkFree_) {
```

Comment:
I mentioned this in the last line in the description

> I also changed initializeRow in RowContainer to always 0 out the memory in the row. Otherwise we could end up with an uninitialized non-empty StringView in the row in the case I described above and still hit a SIGSEGV (I encountered this in the unit test I wrote).